components/common/hooks/useGetTotalUsersNoBots.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useEffect, useState} from 'react';
+
+import {Client4} from 'mattermost-redux/client';
+
+const useGetTotalUsersNoBots = (includeInactive = false): number => {
+    const [userCount, setUserCount] = useState<number>(0);
+
+    const getTotalUsers = async () => {
+        const data = await Client4.getFilteredUsersStats({include_bots: false, include_deleted: includeInactive});
```

Comment:
Would it also make sense to save this data to redux via

```ts
        dispatch({
            type: UserTypes.RECEIVED_FILTERED_USER_STATS,
            data: stats,
        });
```

That would also let us do something where we try to read from redux, and only make the network request if the data isn't yet stored there. Doing that rests on an assumption that slightly stale data is ok, which I *think* for  all these usages in this PR it is. I do see that different usages include deleted users in the count, which would be an argument against storing/reading from redux.

Reply:
IMO, 3/5 I don't think update the store is a good idea for what you mention and for the potential changes in the count of users, so it is better to get fresh data.